std/algorithm/setops.d

```diff
@@ -857,6 +857,46 @@ NWayUnion!(less, RangeOfRanges) nWayUnion
 }
 
 /**
+Computes the intersection of multiple ranges. The input ranges are passed
+as a range of ranges and each is assumed to be sorted by $(D
+less). Computation is done lazily, one intersection element at a time.
+The `multiwayUnion` function is implemented by piping the result
+of $(LREF nWayUnion) through $(REF uniq std,algorithm,iteration).
```

Comment:
Delete this, we don't need to make such guarantees of implementation. Just say "`multiwayUnion(ror)` is functionally equivalent to `multiwayMerge(ror).uniq`".